### PR TITLE
Elaborate datamodel

### PR DESCRIPTION
**Pull request by khannoori**

### CODE REVIEW

File: doc/datamodel.md [hunk 1]
```diff
@@ -6,27 +6,63 @@ The following diagram presents the datamodel behind the Fiziq app.
 In the following, the concepts presented in the above diagram are ellaborated.
 
 ### User
-This concept represents any end-user of the app and does not need to be 
+This concept represents any end-user of the app and does not need to be
 ellaborated. A user has a [TrainingJournal](#TrainingJournal).
 
+
+| Property Name | Type         | Remarks                                    |
```
Comment (kanafghan) [inline doc/datamodel.md]: Please fix the alignment so it looks like a table. I mean like this:

``` markdown
| Property Name            | Type                    | Remarks                          |
|--------------------------|:-----------------------:|----------------------------------|
| name                     | `string`                |                                  |
...
```

File: doc/datamodel.md [hunk 2]
```diff
@@ -6,27 +6,63 @@ The following diagram presents the datamodel behind the Fiziq app.
...
 ellaborated. A user has a [TrainingJournal](#TrainingJournal).
 
+
+| Property Name | Type         | Remarks                                    |
+|----------------------|:-------------:|-------------------------------------------|
+| name                | `string`      |                                                    |
+| email                | `string`      |                                                    |
+| createdAt         | `datetime` |                                                    |
+| updatedAt        | `datetime` |                                                    |
+| TrainingJournal        | `TrainingJournal` |                                                    |
```
Comment (kanafghan) [inline doc/datamodel.md]: the property name, e.g. name, always begins with a lower case letter, thus it is trainingJournal and not TrainingJournal.

File: doc/datamodel.md [hunk 3]
```diff
@@ -6,27 +6,63 @@ The following diagram presents the datamodel behind the Fiziq app.
...
+
 ### TrainingJournal
 This concept represents the log over all trainings sessions performed by a user.
 
+| Property Name | Type         | Remarks                                    |
+|----------------------|:-------------:|-------------------------------------------|
+| name                | `string`      |                                                    |
+| User         | `User` |                                                    |
```
Comment (kanafghan) [inline doc/datamodel.md]: It is user and not User.

File: doc/datamodel.md [hunk 4]
```diff
@@ -6,27 +6,63 @@ The following diagram presents the datamodel behind the Fiziq app.
...
+
 ### WorkoutSession
-This concept represents a period of time where a number of physiqual exercises 
+This concept represents a period of time where a number of physiqual exercises
 are performed by a person.
 
+| Property Name | Type         | Remarks                                    |
+|----------------------|:-------------:|-------------------------------------------|
+| name                | `string`      |                                                    |
+| WorkoutSession         | `WorkoutSession` |                                                    |
```
Comment (kanafghan) [inline doc/datamodel.md]: It is workoutSession and not WorkoutSession.

File: doc/datamodel.md [hunk 5]
```diff
@@ -6,27 +6,64 @@ The following diagram presents the datamodel behind the Fiziq app.
...
 
+| Property Name | Type         | Remarks                                    |
+|----------------------|:-------------:|-------------------------------------------|
+| name                | `string`      |                                                    |
+| MuscleGroup         | `MuscleGroup` |                                                    |
+
 ### MuscleGroup
 There are 7 major muscle groups that all workouts are divided into:
 
+| Property Name | Type         | Remarks                                    |
```
Comment (kanafghan) [inline doc/datamodel.md]: Please put this table below the list.

File: doc/datamodel.md [hunk 6]
```diff
@@ -6,27 +6,63 @@ The following diagram presents the datamodel behind the Fiziq app.
...
+| trainingJournal      | `TrainingJournal` |                                 |
+
+
 ### TrainingJournal
 This concept represents the log over all trainings sessions performed by a user.
 
+TrainingJournal has the following properties:
+| Property Name        | Type          | Remarks                             |
+|----------------------|:-------------:|-------------------------------------|
+| createdAt            | `datetime`        |                                 |
```
Comment (toin0u) [inline doc/datamodel.md]: There is too much spaces after `datetime` :)

**Comment by toin0u** [comment]
Looks good otherwise :+1: 
Remember to sqaush your commits afterwards :)

File: doc/datamodel.md [hunk 7]
```diff
@@ -6,27 +6,63 @@ The following diagram presents the datamodel behind the Fiziq app.
...
 ### WorkoutSet
-This concept consists of a workout/exercise, the number of time it is repeated 
+This concept consists of a workout/exercise, the number of time it is repeated
 and the weights/intensity used to perform it each time.
 
+WorkSet has the following properties
+| Property Name        | Type             | Remarks                          |
+|----------------------|:----------------:|----------------------------------|
+| name                 | `string`         |                                  |
+| Repetition                | `int`       |                                  |
```
Comment (kanafghan) [inline doc/datamodel.md]: Please fix the white spaces here.

File: doc/datamodel.md [hunk 8]
```diff
@@ -6,27 +6,63 @@ The following diagram presents the datamodel behind the Fiziq app.
...
+| Weight               | `float`          |                                  |
+| workoutSession       | `WorkoutSession` |                                  |
 ### Workout
```
Comment (kanafghan) [inline doc/datamodel.md:55]: Insert an empty line before `### Workout`.

File: doc/datamodel.md [hunk 9]
```diff
@@ -6,27 +6,63 @@ The following diagram presents the datamodel behind the Fiziq app.
...
-This concept represents a physiqual exercise that targets one or multiple 
+This concept represents a physiqual exercise that targets one or multiple
 muscle groups and can be performed by a person.
 
+Workout has the following properties
+| Property Name        | Type          | Remarks                             |
+|----------------------|:-------------:|-------------------------------------|
+| name                 | `string`      |                                     |
+| MuscleGroup          | `MuscleGroup` |                                     |
```
Comment (kanafghan) [inline doc/datamodel.md]: property name should be `muscleGroup`.

**Comment by kanafghan** [comment]
@khannoori What is the status on this one?

**Comment by kanafghan** [comment]
:+1:  Khan

**Pull request merged**